app/models.py

```
@@ -0,0 +1 @@
+from . import db
```

Ruff (0.12.2)

`.db` imported but unused

Remove unused import: `.db`

(F401)